syspurpose/src/syspurpose/cli.py

```diff
@@ -348,6 +350,8 @@ def main():
         print(_("WARNING: Setting syspurpose in containers has no effect.\n"
               "Please run syspurpose on the host.\n"))
 
+    print(_("The 'syspurpose' command is deprecated and will be removed in a future major release."
+        " Please use the 'subscription-manager syspurpose' command going forward."), file=sys.stderr)
```

Comment:
I cannot see this message in `./syspurpose/po/keys.pot`. Thus it cannot be translated for this branch.

Reply:
We can update the translation catalogs afterwards, as separate change, like we did for other changes that affected the translatable messages. I will take care of it.

Reply:
OK